docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,52 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
+endif::[]
+
+[[log-correlation]]
+== Log correlation
+
+Log correlation allows you to navigate to all logs belonging to a particular trace and vice-versa:
+for a specific log, see in which context it has been logged and which parameters the user provided.
+
+[float]
+[[log-correlation-ingest]]
+=== Step 1: Ingest your application logs into Elasticsearch
+
+To get started with log correlation, your logs have to be ingested into Elasticsearch first.
+The easiest way to get started with that is https://github.com/elastic/java-ecs-logging[Java ECS logging] which provides layouts/encoders
+for Logback, log4j2, and log4j.
+These layouts/encoders convert a log event into an ECS-compatible JSON structure.
+Using minimal Filebeat configuration,
+you can ingest these logs into Elasticsearch.
+
+Head over to the documentation to learn more: https://github.com/elastic/java-ecs-logging[Java ECS logging documentation].
```

Comment:
It's linking to the same URL as before. Just want to make sure this is intentional.

Reply:
It is intentional. There are no docs on elastic.co yet.